scylla/src/frame/response/result.rs

```diff
@@ -593,7 +585,7 @@ fn deser_rows(buf: &mut &[u8]) -> StdResult<Rows, ParseError> {
     })
 }
 
-#[allow(clippy::unnecessary_wraps)]
+#[allow(clippy::unnecessary_unwrap)]
```

Comment:
Clippy does not complain here about [`unnecessary_unwrap`](https://rust-lang.github.io/rust-clippy/master/#unnecessary_unwrap), but [`unnecesary_wraps`](https://rust-lang.github.io/rust-clippy/master/#unnecessary_wraps). A similar name, but a different issue. Please change the allowed lint name back.

Please also do it for `deser_schema_change`.

Reply:
so sorry!
locally, clippy complained that ` unknown clippy lint: clippy::unnecessary_wraps`, so I just automatically corrected it - gonna change the lint name back rn